x-pack/solutions/search/plugins/search_indices/public/code_examples/javascript.ts

```diff
@@ -149,3 +149,41 @@ const updateMappingResponse = await client.indices.putMapping({
 });
 console.log(updateMappingResponse);`,
 };
+
+export const JSSemanticIngestDataExample: IngestDataCodeDefinition = {
+  ...JSIngestDataExample,
+  ingestCommand: ({
+    apiKey,
+    elasticsearchURL,
+    sampleDocuments,
+    indexName,
+  }) => `const { Client } = require('@elastic/elasticsearch');
+
+const client = new Client({
```

Comment:
@JoshMock while working on this I noticed the client now has an option for `serverMode: 'serverless'` I think it's worth updating these examples to show setting that when in serverless correct?

Reply:
Correct. It updates some defaults to optimize for the serverless environment.